Makefile

```diff
@@ -1,121 +1,99 @@
-# Basys3 board using SymbiFlow, making it simple to understand
-# what to run with a straight-forward Makefile.
-#
-# (c) Henner Zeller <hzeller@google.com>
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Set the environment variable SYMBIFLOW_ARCH_DEFS
-# to wherever the toplevel of your symbiflow-arch-defs
-# is checked out.
-#
-# Call
-#  make prog
-# To synthesize for Basys3 board.
-#
-# Current binaries needed in the process point to the versions
-# checked out via conda in symbiflow, but feel free to
-# set the following environment variables to run your own
-#  - YOSYS_BIN
-#  - VPR_BIN
-#  - GENFASM_BIN
-#  - OPENOCD_BIN
-# to point to local versions of these binaries.
-##
-
-SHELL := /bin/bash
-
-PROJECT=counter
-
 # Point this where symbiflow-arch-defs is checked out
-SYMBIFLOW_ARCH_DEFS?=~/src/symbiflow-arch-defs
+SYMBIFLOW_ARCH_DEFS?=/opt/fpga/symbiflow-arch-defs/
```

Comment:
I'd keep this at the default. Users can change that to wherever they want with the environment variable, but ~/src seems easier to understand.

Reply:
(solved)